shared_model/backend/protobuf/queries/proto_tx_pagination_meta.hpp

```diff
@@ -0,0 +1,35 @@
+/**
+ * Copyright Soramitsu Co., Ltd. All Rights Reserved.
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+#ifndef IROHA_SHARED_PROTO_MODEL_QUERY_TX_PAGINATION_META_HPP
+#define IROHA_SHARED_PROTO_MODEL_QUERY_TX_PAGINATION_META_HPP
```

Comment:
Include guards in other files in this folder are called `IROHA_SHARED_MODEL_PROTO_...`. Please, rename this one according to the common scheme